### PR TITLE
Added .DS_STORE to .gitignore

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,7 @@ coverage/
 /tmp/
 rspec_logs.json
 test_output/
+.DS_STORE
```
Comment (ohwutup) [inline .gitignore]: Should this be, or should we also have a line that ignores the `.DS_Store` (not all caps) spelling as well?

**Comment by ohwutup** [comment]
one little question but otherwise 👍

**Comment by KrauseFx** [comment]
Good question. As far as I know git is not case sensitive, right?

**Comment by ohwutup** [comment]
Not sure about case sensitivity, but may as well merge this guy and see how it works down the line!

**Comment by ohwutup** [comment]
👍

**Pull request merged**